lp/ui/views.py

```diff
@@ -501,7 +501,8 @@ def search(request):
         if q:
             q += " AND"
         q += " lccallnum:('gw electronic' OR 'shared+electronic'" + \
-            " OR 'e-resources' OR 'e-govpub' OR 'streaming')"
+            " OR 'e-resources' OR 'e-govpub' OR 'streaming'" + \
+	    " OR 'hathitrust)"
```

Comment:
Looks like a single quote is missing from after hathitrust. Works, but probably not ideal.